lib/utils/check-version.js

```diff
@@ -0,0 +1,41 @@
+const request = require("request-promise");
+const Logger = require.main.require("lib/logger/logger.js");
+const Errors = require.main.require("lib/response/error.js");
+
+function CheckVersion(pkg) {
+    if (typeof pkg === "object") {
+        const pkg_split = pkg.version.split(".");
+        const version = pkg_split[0] + "." + pkg_split[1];
+        const url = "http://registry.npmjs.org/sealious/" + version;
+        let status = "warning", message;
```

Comment:
I find the `let status = "warning", message;` very unintuitive, i think it's easier to see how many variables you're creating when each of them is preceded by `let`. I understand that the syntax you've used here is much shorter, but I think that in the back-end code especially we should favor readability over brevity.